tests/test_dse.py

```diff
@@ -2493,6 +2493,24 @@ def test_hoisting_pow_one(self):
 
         assert len([i for i in FindSymbols().visit(op) if i.is_Array]) == 0
 
+    def test_zeroable_property(self):
+        grid = Grid(shape=(10, 10))
+
+        u = TimeFunction(name='u', grid=grid, space_order=4)
+
+        op = Operator(Eq(u.forward, u.dx.dx))
+        iterations = FindNodes(Iteration).visit(op)
+        v = iterations.pop(2)
+        assert ZEROABLE in v.properties
+        assert not any(ZEROABLE in i.properties for i in iterations)
+
+        # Fusion makes ZEROABLE disappear
+        for eq in [Eq(u.forward, u.dx.dx + u.dy.dy),
+                   Eq(u.forward, u.dx.dx + u.dx.dy)]:
+            op = Operator(eq)
```

Comment:
I would recommend to op.apply() as well.

Reply:
testing for what?

Reply:
It reminded me of an issue I had with fcc compiler. In case the loop was going from 0 to 0 openmp was segfaulting.
We should be safe for what we have now, but it would not harm I guess. In any case it is nitpicking

Reply:
I totally get it. I am just wondering whether there is a chance that the loop goes from 0 to 0.